installation/rhel-setup.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash
+
+#  This file is part of cc-runtime.
+#
+#  Copyright (C) 2018 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+# Description: This script installs Clear Containers on a RHEL system.
+
+# enable tracing
+set -x
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/installation-setup.sh"
+source "${SCRIPT_PATH}/../versions.txt"
+
+# List of packages to install to satisfy build dependencies
+pkgs=""
+
+# general
+pkgs+=" bc"
+pkgs+=" curl"
+
+# qemu lite dependencies
+pkgs+=" libattr-devel"
+pkgs+=" libcap-devel"
+pkgs+=" libcap-ng-devel"
+pkgs+=" pixman-devel"
+pkgs+=" glib2-devel"
+pkgs+=" zlib-devel"
+
+devtoolset_version="7"
```

Comment:
Could you add this variable (as `rhel_devtoolset_version`) and `required_gcc_version=` (as `rhel_required_gcc_version=`) to the top-level `./versions.txt`?